### PR TITLE
Arcade VSCode extension blog post

### LINKED ISSUE #9445: MakeCode Arcade extension blog post
Hi Galen,
Can you post this up to the blog when you get a chance?
Files here: https://1drv.ms/f/s!AmMIW5Hxi0RtgaEtXg_qVFXt3xmhbA?e=LZYCB4 
Thanks!
Jacqueline

### PR DESCRIPTION
**Pull request by ganicke**
Import, format, edit, adjust images, etc. for the MakeCode Arcade VS Code extension blog post.

Closes #9445

### CODE REVIEW

File: docs/blog/arcade/vscode-extension.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+* Are comfortable writing your MakeCode Arcade games in Typescript/JavaScript.
+* Would like access to full language support and intellisense.
+* Have a complex game that you might want to split up into multiple code files.
+* Want to build skills using a professional development tool.
+* Are itching for richer editing tools like syntax highlighting, bracket-matching, auto-indentation, keyboard shortcut mappings and refactoring tools.
+* Want to use [Live Share](https://visualstudio.microsoft.com/services/live-share) or other VS Code extensions.
+
+Here's a look at a game project opened in the MakeCode Arcade Extension:
+
+![MakeCode Arcade extension at work](/static/blog/arcade/vscode-extension/extension.png)
```
Comment (ganicke) [inline docs/blog/arcade/vscode-extension.md:20]: I added the project in the extension image you left out, if you want it in there.

File: docs/blog/arcade/vscode-extension.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+Check out the brief video tour:
+
+https://youtu.be/R-s-ro8qTUs
+
+We are shipping this as a "Preview" because we know there are some improvements we want to make, including an online version with [vscode.dev](https://insiders.vscode.dev/makecode), support for non-named assets, debugging and a bunch of other fixes. To get started, see this brief [setup page](https://arcade.makecode.com/vscode).
```
Comment (ganicke) [inline docs/blog/arcade/vscode-extension.md:26]: Do you want to expose the actual vscode.dev link at this point?

File: docs/blog/arcade/vscode-extension.md (hunk 1 above)
Comment (Jaqster) [reply docs/blog/arcade/vscode-extension.md:20]: Looks great!  Thanks!

File: docs/blog/arcade/vscode-extension.md (hunk 2 above)
Comment (Jaqster) [reply docs/blog/arcade/vscode-extension.md:26]: No, let's not point to the vscode.dev route just yet.  It's not quite ready...

Comment (Jaqster) [reply docs/blog/arcade/vscode-extension.md:26]: I just realized that the video I uploaded isn't cropped correctly (black space on the left).  I just uploaded a new video - can you point to this one?  https://youtu.be/EpHsJfztiPU

**Review by Jaqster — APPROVED** [review]
Looks good!  Can you just switch out the video link?

**Pull request merged**